consensus/enclave/mock/src/lib.rs

```diff
@@ -216,67 +288,61 @@ impl ConsensusEnclave for ConsensusServiceMockEnclave {
         &self,
         parent_block: &Block,
         inputs: FormBlockInputs,
-        _root_element: &TxOutMembershipElement,
+        root_element: &TxOutMembershipElement,
     ) -> Result<(Block, BlockContents, BlockSignature)> {
-        let block_version = self.blockchain_config.lock().unwrap().block_version;
-        let transactions_with_proofs: Vec<(Tx, Vec<TxOutMembershipProof>)> = inputs
-            .well_formed_encrypted_txs_with_proofs
-            .iter()
-            .map(|(encrypted_tx, proofs)| {
-                // These bytes are normally an enclave-encrypted Tx, but here, it is just
-                // serialized.
-                let ciphertext = &encrypted_tx.0;
-                let tx = mc_util_serial::decode::<Tx>(ciphertext).unwrap();
-                (tx, proofs.clone())
-            })
-            .collect();
-
-        // root_elements contains the root hash of the Merkle tree of all TxOuts in the
-        // ledger that were used to validate the tranasctions.
-        let mut root_elements = Vec::new();
         let mut rng = McRng::default();
+        let config = self.blockchain_config.lock().unwrap();
 
-        for (tx, proofs) in transactions_with_proofs.iter() {
-            mc_transaction_core::validation::validate(
-                tx,
-                parent_block.index + 1,
-                block_version,
-                proofs,
-                Mob::MINIMUM_FEE,
-                &mut rng,
-            )?;
-
-            for proof in proofs {
-                let root_element = compute_implied_merkle_root(proof)
-                    .map_err(|_e| TransactionValidationError::InvalidLedgerContext)?;
-                root_elements.push(root_element.clone());
-            }
-        }
```

Comment:
Would removing this have any side effects on other tests?

Reply:
It wasn't removed, it moved up. See lines 126-130.